cloudwatch/cloudwatch.go

```diff
@@ -193,6 +214,9 @@ func NewOutputPlugin(config OutputPluginConfig) (*OutputPlugin, error) {
 		logGroupRetention:             config.LogRetentionDays,
 		autoCreateGroup:               config.AutoCreateGroup,
 		groups:                        make(map[string]struct{}),
+		ecsMetadata:                   TaskMetadata{},
+		runningInECS:                  runningInECS,
+		uuid:                          ksuid.New().String(),
```

Comment:
just curious- how many characters is it

Reply:
The format is like 4 bytes of time (seconds) + 16 random bytes. And here is an example: 1jD7P6bbSRtbc9stkWjJZYerO6s.